third_party/react-dates/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "react-dates",
+  "version": "0.0.0",
+  "description": "Bundles react-dates so that amp-date-picker can use it.",
+  "main": "index.js",
+  "author": "The AMP HTML Authors",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ampproject/amphtml.git"
+  },
+  "devDependencies": {
+    "acorn-globals": "^6.0.0",
+    "ast-replace": "^1.1.3",
+    "babelify": "^10.0.0",
+    "browserify": "^17.0.0",
+    "derequire": "^2.1.1",
+    "envify": "^4.1.0",
+    "escodegen": "^2.0.0",
+    "prop-types": "^15.7.2",
+    "react-addons-shallow-compare": "^15.6.3",
+    "react-externs": "^0.13.6",
+    "react-with-direction": "^1.3.1",
+    "rocambole": "^0.7.0",
+    "uglifyify": "^5.0.2"
+  }
```

Comment:
Can we use exact versions here to streamline renovate updates? (Removing the carets should suffice.) At that point, I believe these packages will get grouped under `subpackage devDependencies`:

https://github.com/ampproject/amphtml/blob/aa3c1a04d106ae70615eb2ee70e2c1d34a633e5f/.renovaterc.json#L24-L31